libs/components/help-inline/src/lib/modules/help-inline/help-inline.component.html

```diff
@@ -21,19 +21,14 @@
   [skyPopover]="popoverContent ? popover : undefined"
   (click)="onClick()"
 >
-  <sky-icon *skyThemeIf="'default'" icon="info-circle" />
-  <sky-icon-stack
-    *skyThemeIf="'modern'"
-    size="xs"
-    [baseIcon]="{
-      icon: 'circle-solid',
-      iconType: 'skyux'
-    }"
-    [topIcon]="{
-      icon: 'help-i',
-      iconType: 'skyux'
-    }"
-  />
+  <i *skyThemeIf="'default'" class="sky-icon fa fa-info-circle"></i>
+  <span *skyThemeIf="'modern'" aria-hidden="true" class="help-inline-icon"
+    ><span class="sky-icon-stack fa-stack fa-xs sky-icon-stack-size-xs"
+      ><i class="sky-icon fa-stack-2x sky-i-circle-solid"></i
+      ><i
+        class="sky-icon fa-stack-1x fa-inverse sky-icon-inverse sky-i-help-i"
```

Comment:
what is that last class that is added? `sky-i-help-i`?

Reply:
That is the help icon from [`skyux-icons.css`](https://github.com/blackbaud/skyux-icons/blob/master/src/css/skyux-icons.css#L153).